js/indicators/tema.src.js

```diff
@@ -31,30 +31,56 @@ H.seriesType('tema', 'ema',
      * @optionparent plotOptions.tema
      */
     {}, {
+        getEMA: function (
+            yVal,
+            prevEMA,
+            SMA,
+            index,
+            i,
+            xVal
+          ) {
+            index = index === undefined ? -1 : index;
+            i = i === undefined ? 1 : i;
+            xVal = xVal || [];
+
+            return EMAindicator.prototype.calculateEma(
+                xVal,
+                yVal,
+                i,
+                this.chart.series[0].EMApercent,
+                prevEMA,
+                index,
+                SMA
+            );
```

Comment:
Use arguments as declared values:

```
return EMAindicator.prototype.calculateEma(
                xVal || [],
                yVal,
                i === undefined ? 1 : i,
                this.chart.series[0].EMApercent,
                prevEMA,
                index === undefined ? -1 : index,
                SMA
            );
```
and remove lines 42-44

Reply:
All changes have been made.